docker-compose.yml

```diff
@@ -12,14 +12,15 @@ services:
     ports:
       - "8080:8080"
     environment:
-      CFG_DATABASE_URL: postgres://postgres:postgres@postgres:5432/config_db
-      CFG_LISTEN: 0.0.0.0:8080
-      CFG_METRICS_ENDPOINT: 0.0.0.0:19000
-      CFG_FOLLOWER_URL: "http://node:8090"
-      CFG_NETWORK: mainnet
-      CFG_ADMIN: ${CONFIG_ADMIN_PUBKEY}
-      CFG_KEYPAIR: /config-signing-key.bin
-      CFG_LOG: info
+      CFG__DATABASE__URL: postgres://postgres:postgres@postgres:5432/config_db
+      CFG__DATABASE__MAX_CONNECTIONS: 50
```

Comment:
Did you want this to be double underscore as well? `CFG__DATABASE__MAX__CONNECTIONS`?

Reply:
in this case you wouldn't; the double is _only_ between the levels of an application/crate whereas the single underscore is just part of the setting name